server/src/lib/model/user-client.ts

```diff
@@ -191,12 +191,14 @@ const UserClient = {
     await Promise.all([
       this.claimContributions(accountClientId, clientIds),
       locales && updateLocales(accountClientId, locales),
-      this.enrollRegisteredUser(
-        email,
-        data.enrollment.challenge,
-        data.enrollment.team,
-        data.enrollment.invite
-      ),
+      data &&
+        data.enrollment &&
+        this.enrollRegisteredUser(
+          email,
+          data.enrollment.challenge,
+          data.enrollment.team,
+          data.enrollment.invite
+        ),
```

Comment:
This seems like a more pressing fix, so I'm going to pull it into its own commit.

Reply:
https://github.com/mozilla/voice-web/commit/2f59870385a4fe460ed2cebeb08c71f76d32b8dd